velox/exec/TableWriter.h

```diff
@@ -140,6 +140,19 @@ class TableWriter : public Operator {
     return stats;
   }
 
+  /// The name of runtime stats specific to table writer.
+  /// The running wall time of a writer operator from creation to close.
+  static inline const std::string kRunningWallNanos{"runningWallNanos"};
+  /// The number of files written by this writer operator.
+  static inline const std::string kNumWrittenFiles{"numWrittenFiles"};
+  /// The file write IO walltime.
+  static inline const std::string kWriteIOTime{"writeIOWallNanos"};
+  /// The walltime spend on file write data recoding.
+  static inline const std::string kWriteRecodeTime{"writeRecodeWallNanos"};
+  /// The walltime spent on file write data compression.
+  static inline const std::string kWriteCompressionTime{
```

Comment:
Sorry to bother you.

I'm currently learning Velox and working with its metrics system. I noticed that the two metrics(`kWriteRecodeTime` & `kWriteCompressionTime`) updated in this PR don't seem to be updating for me; their values are consistently 0.

Could you please provide some insight into why this might be happening? Am I perhaps missing a specific configuration or step required for these metrics to be populated?

Thanks for your help!